Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -19,6 +19,11 @@ public enum NotificationAuthorizationStatus {
   case provisional
 }
 
+public enum KSROptimizelyLogLevel {
```

Comment:
Any reason you chose this instead of `OptimizelyLogLevelType` in its own file? It seems similar to `OptimizelyClientType` and `OptimizelyResultType`.

Reply:
I agree `OptimizelyLogLevelType` sounds better 👍